images/virt-artifact/patches/024-cover-kubevirt-metrics.patch

```diff
@@ -0,0 +1,165 @@
+diff --git a/cmd/virt-handler/virt-handler.go b/cmd/virt-handler/virt-handler.go
+index 09b2b53904..c46122539c 100644
+--- a/cmd/virt-handler/virt-handler.go
++++ b/cmd/virt-handler/virt-handler.go
+@@ -93,10 +93,12 @@ const (
+ 	defaultWatchdogTimeout = 30 * time.Second
+ 
+ 	// Default port that virt-handler listens on.
+-	defaultPort = 8185
++	defaultPort        = 8185
++	defaultMetricsPort = defaultPort
```

Comment:
Set other number as default. For example
```suggestion
+	defaultMetricsPort = 8186
```

Reply:
set it to 8080